express-static/public/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <title>Express Static</title>
+  </head>
+  <body>
+    <h1>Static Malware</h1>
```

Comment:
Try putting an image tag in here so you can experiment with how image paths are resolved.
To do that, you will also need to create `public/images` and add some images to it.